catalog-info.yaml

```
@@ -118,6 +118,11 @@ spec:
           access_level: MANAGE_BUILD_AND_READ
         everyone:
           access_level: READ_ONLY
+      schedules:
+        Daily 7.17:
+          branch: "7.17"
+          cronline: "@daily"
+          message: Builds daily `7.17` dra
```

https://buildkite.com/docs/pipelines/configure/workflows/scheduled-builds#schedule-intervals-predefined-intervals

```suggestion
        Weekly 7.17:
          branch: "7.17"
          cronline: "@weekly"
          message: Builds `7.17` DRA
```